src/qibolab/platform.py

```diff
@@ -30,88 +39,57 @@ def __init__(self, name, runcard, instruments, channels):
         log.info("Loading platform %s", name)
 
         self.name = name
-        self.runcard = runcard
+        self.is_connected = False
         self.instruments: List[Instrument] = instruments
         self.channels: ChannelMap = channels
 
-        self.qubits: Dict[QubitId, Qubit] = {}
-        self.pairs: Dict[Tuple[QubitId, QubitId], QubitPair] = {}
-
-        # Values for the following are set from the runcard in ``reload_settings``
-        self.settings = None
-        self.is_connected = False
-
-        self.nqubits = None
-        self.resonator_type = None
-        self.topology = None
-
-        self.nshots = None
-        self.relaxation_time = None
-        self.sampling_rate = None
-
-        # TODO: Remove this (needed for the multiqubit platform)
-        self.native_gates = {}
-        self.two_qubit_native_types = NativeType(0)
-        # Load platform settings
-        self.reload_settings()
-
-    def __repr__(self):
-        return self.name
-
-    def _check_connected(self):
-        if not self.is_connected:  # pragma: no cover
-            raise_error(RuntimeError, "Cannot access instrument because it is not connected.")
-
-    def reload_settings(self):
-        # TODO: Remove ``self.settings``
-        if self.settings is None:
-            # Load initial configuration
-            if isinstance(self.runcard, dict):
-                settings = self.settings = self.runcard
-            else:
-                with open(self.runcard) as file:
-                    settings = self.settings = yaml.safe_load(file)
+        # Load initial configuration
+        if isinstance(runcard, dict):
+            settings = runcard
         else:
-            # Load current configuration
-            settings = self.settings
+            with open(runcard) as file:
+                settings = yaml.safe_load(file)
```

Comment:
If you want to do this passage in a single line you can do something like this https://github.com/qiboteam/qibocal/pull/372#discussion_r1227989554

Reply:
Actually never mind, my suggestion was for dumping not loading. You can ignore this comment.

Reply:
It works just the same for loading, it's just:
```suggestion
            settings = yaml.safe_load(runcard.read_text())
```

Reply:
Fixed this, thanks.